portfolio/pom.xml

```diff
@@ -18,10 +18,22 @@
 
   <dependencies>
     <dependency>
-      <groupId>javax.servlet</groupId>
-      <artifactId>javax.servlet-api</artifactId>
-      <version>4.0.1</version>
-      <scope>provided</scope>
+        <groupId>javax.servlet</groupId>
```

Comment:
Prefer to keep the same indent through the entire file. For example the previous content has 2 spaces indent while here has 4 spaces.

Reply:
Fixed the indentation!